.github/workflows/composer.test.yml

```diff
@@ -44,3 +44,6 @@ jobs:
       run: docker-compose exec -T api vendor/bin/phpunit
     - name: Psalm
       run: docker-compose exec -T api vendor/bin/psalm
+
+    - name: Run blazegraph integration test
+      run: docker-compose exec -e RUN_PHPUNIT_INTEGRATION_TEST=1 -T api vendor/bin/phpunit tests/Jobs/Integration/QueryserviceNamespaceJobTest.php
```

Comment:
Make make sense to make this more generic and just pass the path of `tests/Jobs/Integration` and name this `Run integration tests` or something like that?
But maybe to do that properly the test dir would need a bit of a restructure?
Could probably leave this for later...